chia/daemon/server.py

```diff
@@ -1155,7 +1155,10 @@ async def exit(self) -> None:
 
     async def register_service(self, websocket: WebSocketResponse, request: Dict[str, Any]) -> Dict[str, Any]:
         self.log.info(f"Register service {request}")
-        service = request["service"]
+        service = request.get("service")
+        if service is None:
+            self.log.error("cant register, missing service name")
```

Comment:
`"Service Name missing from request to 'register_service'"` would provide a little more information